src/setfit/data.py

```diff
@@ -222,7 +225,7 @@ class SetFitDataset(TorchDataset):
     Args:
         x (`List[str]`):
             A list of input data as texts that will be fed into `SetFitModel`.
-        y (`List[int]`):
+        y (`Union[List[int], List[List[int]]]`):
```

Comment:
@blakechi I didn't really understand which changes to the dataset you were thinking about because it works as is, but I updated the type hints to make it more obvious that it works with multitarget labels.

Reply:
Oh sorry, I should be more specific. I was thinking the converting for labels, which I saw you added this at line 248 and 249.
Look great now :)